app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All friends should be vaccinated")
+        today = datetime.date.today()
+        expiration_date = visitor["vaccine"]["expiration_date"]
+        if today > expiration_date:
```

Comment:
There is no need to create variables since they are used only once in the code. Evaluate the expression as is without variable assignment in the future if possible.